ecs-task-definition-templates/deployment-mode/sidecar/combination/combination-ec2.json

```diff
@@ -22,7 +22,7 @@
         },
         {
             "name": "cloudwatch-agent",
-            "image": "amazon/cloudwatch-agent:latest",
+            "image": "public.ecr.public/cloudwatch-agent/cloudwatch-agent:latest",
```

Comment:
This is broken... guessing a sed gone wrong somewhere?

Reply:
`public.ecr.aws/cloudwatch-agent/cloudwatch-agent:latest` is what I see from the ECR gallery. I should have caught this.